test/handshake_helper.c

```diff
@@ -637,7 +637,8 @@ static int configure_handshake_ctx(SSL_CTX *server_ctx, SSL_CTX *server2_ctx,
     }
     if (extra->client.alpn_protocols != NULL) {
         unsigned char *alpn_protos = NULL;
-        size_t alpn_protos_len;
+        size_t alpn_protos_len = 0;
```

Comment:
really that should be another PR, and probably also for master

Reply:
Ok, I've removed it here and will add separate PR(s). 
The one Travis failure resulting from that issue can be ignored for this PR.
Can you please re-confirm?